pkgs/pkgs-lib/formats/hocon/test/backwards-compatibility/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib, formats, stdenvNoCC, writeText, ... }:
```

Comment:
lib and writeText seem to be unused

Reply:
yes, within nixpkgs and within this pr there is a lot of dead code and unused dependencies. I removed a few that I stumbled across putting this together, but I'm certain that there are a lot more instances of this